managers/templatetags/custom_tags.py

```diff
@@ -0,0 +1,38 @@
+from django import template
```

Comment:
Template tags with database queries should only be used when necessary. At best - never :) All data required by the template frontend should be prepared in backend, to keep template frontend logic at minimum. The same effect can be achieved by using Django custom model manager, but it has advantage that it can be used in both template and Python code, so there is no need to duplicate this functionality.

Definitely `filter_terminated`, `filter_active` and `filter_completed` can be moved as custom manager methods because they don't accept any extra parameters. About the rest I can't tell because they are not used in this pull request, but I think it is doable to, so please do them all as custom manager methods.

Please check https://simpleisbetterthancomplex.com/tips/2016/08/16/django-tip-11-custom-manager-with-chainable-querysets.html for an example how to do this. If you need help please contact me.

Reply:
Done